src/api/cvc4cpp.cpp

```diff
@@ -16,7 +16,11 @@
 
 #include "api/cvc4cpp.h"
 
+#include "base/cvc4_assert.h"
+#include "expr/expr.h"
+#include "expr/expr_manager.h"
 #include "util/result.h"
+#include "util/utility.h"
```

Comment:
One of the Travis builds fails because std::find_if is defined in `<algorithm>` and that include is missing here.

Reply:
It is actually missing in util/utility.h, so I added it there.